clients/common/rocblas_gentest.py

```diff
@@ -221,6 +221,14 @@ def setdefaults(test):
         if all([x in test for x in ('stride_scale')]):
             test.setdefault('stride_c', int(test['stride_scale']) * 5)
 
+    elif test['function'] in ('tpmv_strided_batched'):
+        if all([x in test for x in ('M', 'incx', 'stride_scale')]):
+            ldx = int(test['M'] * abs(test['incx']) * test['stride_scale'])
+            test.setdefault('stride_x', ldx)
+        if all([x in test for x in ('M', 'stride_scale')]):
+            ldM = int( ( (test['M'] + 1) * test['M'] ) / 2 * test['stride_scale'])
+            test.setdefault('stride_a', ldM)
+
```

Comment:
I think Daine has added the setkey_product helper that you may find useful to clean up the code

Reply:
Thanks, will take a look and will adjust.

Reply:
done